tempto-core/src/main/java/io/prestodb/tempto/internal/convention/AnnotatedFileParser.java

```diff
@@ -143,11 +144,11 @@ private int findNextSectionIndex(List<String> lines)
      */
     private List<String> filterContent(List<String> contentLines)
     {
-        contentLines = contentLines.stream()
+        return contentLines.stream()
                 .filter(s -> !(isSpecialLine(s) || isBlank(s)))
+                .map(line -> line.replace(NEW_LINE_LITERAL, "\n"))
```

Comment:
- this definitely needs to go _after_ `.map(AnnotatedFileParser::unescapeLine)`
- can `\n` in the file be escaped by yet another `\`?

Reply:
> can \n in the file be escaped by yet another \?

There is no such thing like escaping characters in data files.

Reply:
so now it won't be possible to have literal `\n` (not `␊`) in the data file, right?

Reply:
right. It is like `\NULL` which stands for `null`.